app/renderer/actions/Session.js

```diff
@@ -869,6 +874,32 @@ export function getRunningSessions () {
   };
 }
 
+export function startDesiredCapsNameEditor () {
+  return (dispatch) => {
+    dispatch({type: ENABLE_DESIRED_CAPS_NAME_EDITOR});
+  };
+}
+
+export function abortDesiredCapsNameEditor () {
+  return (dispatch) => {
+    dispatch({type: ABORT_DESIRED_CAPS_NAME_EDITOR});
+  };
+}
+
+export function saveRawDesiredCapsName () {
```

Comment:
i think the word `raw` here is a bit unnecessary. we use `raw` for the other aspect of the caps editor because the user is editing raw JSON instead of using the UI. but in the case of the title, there is no distinction between raw or non-raw. so i think it's not necessary to make that distinction in code either. i'd just use `saveDesiredCapsName` here instead (and as well in other places where 'raw' is used for the title)